question/app.js

```diff
@@ -0,0 +1,158 @@
+//基礎編
+  //Q1 変数
+  let nickname = 'ごっしー';
+  let age = 28;
+  console.log('私のニックネームは' +nickname+ 'です。年齢は' +age+ 'です。');
```

Comment:
[must]
`+`の前後は半角スペース設けてください！
そのほかにも、インデントやスペースが間違っている部分は修正してください！
```suggestion
  console.log('私のニックネームは' + nickname + 'です。年齢は' + age + 'です。');
```

[ask]
なぜ、インデントやスペースは統一する必要があるのでしょうか？

Reply:
可読性(コードを全体的に見やすくするため)です。
インデントのルールがバラバラだと逆に見えずらくなるので、一つ目の指摘とともに今一度全体的に可読性を確認して修正します。

Reply:
[good]
- 指摘箇所だけでなく類似箇所も確認して修正しているの素晴らしいですね！